nearcore/src/metrics.rs

```diff
@@ -0,0 +1,22 @@
+use near_metrics::{try_create_histogram_vec, HistogramVec};
+use once_cell::sync::Lazy;
+
+pub static APPLY_CHUNK_DELAY: Lazy<HistogramVec> = Lazy::new(|| {
+    try_create_histogram_vec(
+        "near_apply_chunk_delay",
+        "Time process a chunk. Tgas used by the chunk is a metric label.",
+        &["tgas_ceiling"],
+        Some(prometheus::linear_buckets(0.0, 0.05, 50).unwrap()),
+    )
+    .unwrap()
+});
+
+pub static SECS_PER_THOUSAND_TGAS: Lazy<HistogramVec> = Lazy::new(|| {
+    try_create_histogram_vec(
+        "near_execution_secs_per_thousand_tgas",
+        "Execution time per teragas. Ignore label 'label'.",
+        &["label"],
```

Comment:
What is the purpose of this `label`? If it needs to be here, consider adding a comment?

Reply:
Added a comment

Reply:
Huh, that's a curious and needless limitation, which I didn't encounter before. Sounds like there's nothing stopping us from removing the label in the future, though.

Reply:
Actually, let me take that back. An empty list of labels works.